blocks/dashboard/renderAdmin.js

```diff
@@ -1,15 +1,134 @@
-/* eslint-disable no-unused-vars */
-/* eslint-disable no-undef */
 import {
   OOPS,
   SCRIPT_API,
   waitForAuthenticated,
   loadingSpinner,
+  dateToRelativeString,
 } from '../../scripts/scripts.js';
 import { readQueryParams, removeQueryParams, writeQueryParams } from '../../libs/queryParams/queryParams.js';
 import paginator from '../../libs/pagination/pagination.js';
 import renderAnalytics from '../../scripts/analytics.js';
+import { toClassName } from '../../scripts/aem.js';
 
+const langNames = new Intl.DisplayNames(['en'], { type: 'language' });
+function parseAcceptLanguage(str) {
+  if (!str) return null;
+  return langNames.of(str.split(',')[0]);
+}
+
+// header: sec-ch-ua
+function parseBrowser(str) {
+  if (!str) return null;
+
+  const browserParts = str.split(',');
+  try {
+    let output = '';
+    for (const browserStr of browserParts) {
+      const browserName = browserStr.split(';')[0].trim().replaceAll('"', '');
+
+      if (/not\?a_brand/i.test(browserName)) continue;
```

Comment:
<img width="120" alt="Screenshot 2024-10-23 at 15 13 38" src="https://github.com/user-attachments/assets/a106735b-a95f-4a43-80e1-bd211dbb567f">

This is still showing up not sure if intended ?

Reply:
this edgecase is now handled